### PR TITLE
[coopr-645] old UI integration tests removed

### PR DESCRIPTION
**Pull request by kpovetkin**
according to https://issues.cask.co/browse/COOPR-645

### CODE REVIEW

File: coopr-integration-testing/src/test/java/co/cask/coopr/test/Constants.java [hunk 1]
```diff
@@ -1,181 +0,0 @@
...
-import java.net.ServerSocket;
-
-/**
- *
- */
-public class Constants {
-
-  // CLI commands for integration tests
-  // Admin commands
-  public static final String LIST_TEMPLATES_COMMAND = "list templates";
```
Comment (jwang47) [inline coopr-integration-testing/src/test/java/co/cask/coopr/test/Constants.java:31]: Aren't these needed for `AdminCommandsTest`?

**Comment by jwang47** [comment]
Travis build passes b/c it doesn't build the coopr-integration-testing module. Can you verify that the coopr-integration-testing module builds? It appears that it wouldn't since `Constants.LIST_TEMPLATES_COMMAND` is used by `AdminCommandsTest`.

**Comment by kpovetkin** [comment]
Updated.

**Comment by kpovetkin** [comment]
I had a problem with tests:

If I run 'mvn test' I'll get:
" cannot find symbol
[ERROR] symbol:   class HandlerServer
[ERROR] location: class co.cask.coopr.test.client.ClientTest"

There is a workaround that works for me:

Replace all "HandlerServer" occurrences in ClientTest.java to "ExternalHandlerServer" .

Is such a behavior normal?

**Comment by jwang47** [comment]
Build is passing, so merging.

**Pull request merged**